apps/SageAccountWeb/src/style/_AccountCreatedPage.scss

```diff
@@ -1,11 +1,14 @@
 .AccountCreatedPage {
+  .right-panel {
+    margin-top: 16px;
+  }
   button,
   a {
     width: 100%;
     margin: 10px 0px;
     display: block;
   }
-  .sourceAppItem img {
+  .sourceAppItem svg {
```

Comment:
Fix sizing of logo in account created page (see Cancer Complexity Knowledge Portal logo), logo changed to svg in this [commit](https://github.com/Sage-Bionetworks/synapse-web-monorepo/commit/ee3810b058f2dced89e7a86b1aa3173c0880ec2f)